src/index.html

```diff
@@ -8,5 +8,38 @@
   </head>
   <body>
     <h1>Calendar</h1>
+    <div class="calendar">
```

Comment:
You should add modificators to your calendar
![image](https://user-images.githubusercontent.com/30314756/148032185-07d64ab9-0ad7-4b83-b0f8-96eb2508ff04.png)